azure-mgmt-sql/src/test/java/com/microsoft/azure/management/sql/SqlServerOperationsTests.java

```diff
@@ -188,13 +191,13 @@ public void canManageReplicationLinks() throws Exception {
                 .withCreateMode(CreateMode.ONLINE_SECONDARY)
                 .create();
         Thread.sleep(2000);
-        PagedList<ReplicationLink> replicationLinksInDb1 = databaseInServer1.replicationLinks().list();
+        List<ReplicationLink> replicationLinksInDb1 = databaseInServer1.replicationLinks().list();
```

Comment:
actually, i'm wonderering, for any List<T> we may be returning across the fluent API surface that do NOT come from PagedList, but we instantiate the list ourselves instead, are we able to return Set<T> instead?  To answer that question in general, two questions to ask ourselves for each such case:
a) is there any meaning to the order in which the items are arranged in the list?
b) is it ok for more than one instance of the same entry to exist in the list?  (like two instances of the same string, or the same object)

If the answer is No to both, then we should consider returning Set<T> instead.  
I don't know what is the case with the List returning APIs in SQL we have today, so this is just something to think about in general.